ts/receiver/types.ts

```diff
@@ -0,0 +1,16 @@
+import { SignalService } from '../protobuf';
+
+export interface Quote {
+  id: any;
+  author: any;
+  attachments: Array<any>;
+  text: string;
+  referencedMessageNotFound: boolean;
+}
```

Comment:
in ChatMessage.ts  I have a declaration of 
```
export interface Quote {
  id?: number;
  author?: string;
  text?: string;
  attachments?: Array<QuotedAttachment>;
```
with the type description of a QuotedAttachment too. Should those two interface be merged in a single one?

Reply:
Quote has extra fileds on top of what is in the protobuf. Also I think we should use a tighter type for Quote (and others) inside business logic and only deal with protobuf types on the network boundary.